client/jetpack-connect/woo-install-ext-success-notice.tsx

```diff
@@ -0,0 +1,27 @@
+import { removeQueryArgs } from '@wordpress/url';
+import { localize, LocalizeProps } from 'i18n-calypso';
+import page from 'page';
+import Notice from 'calypso/components/notice';
+
+const WooInstallExtSuccessNotice = ( { translate }: LocalizeProps ) => {
+	return (
+		<Notice
+			className="jetpack-connect__woo-core-profiler-notice"
+			status="is-success"
+			showDismiss={ false }
+			text={ translate( 'Extensions successfully installed!' ) }
+			isCompact={ false }
+			duration={ 3000 }
```

Comment:
This feels too short. What do you think?
```suggestion
			duration={ 6000 }
```

Reply:
Sounds good!